wire-ios-request-strategy/Sources/Payloads/Processing/ConversationEventPayloadProcessor.swift

```diff
@@ -687,15 +687,6 @@ final class ConversationEventPayloadProcessor {
             groupID: payload.mlsGroupID,
             context: context
         )
-
-        if source == .slowSync {
-            await context.perform {
-                mlsEventProcessor.joinMLSGroupWhenReady(
-                    forConversation: conversation,
-                    context: context
-                )
-            }
-        }
```

Comment:
not needed anymore?

Reply:
The end result of this call was to add the conversation to the list of `pendingJoins` in the `MLSService`. I removed this property and now we're getting the pending joins via a fetch request instead.